### ISSUE TITLE
Unit tests are needed.

### ISSUE THREAD

**Issue by mariuszstr** [morgante/django-graph-auth#10]
I think unit tests are needed for this library. I can start writing It.

**Comment by morgante** [comment]
Yes, I definitely agree but haven't had time to write them. If you write some I'd be very open to a PR.
